modules/metallb-installing-using-web-console.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-operator-install.adoc
+
+:_content-type: PROCEDURE
+[id="installing-the-metallb-operator-using-web-console_{context}"]
+= Installing the MetalLB Operator from the OperatorHub using the web console
+
+As a cluster administrator, you can install the MetalLB Operator by using the {product-title} web console.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+. Create the required namespace for the MetalLB Operator:
+.. Navigate to *Administration* -> *Namespaces* and click *Create Namespace*.
+.. Enter `metallb-system` in the *Name* field and click *Create*.
```

Comment:
The namespace can also be created when the install is initiated. The user is given ability to select or create the project.

Reply:
Yes I saw that too - not sure of best approach

Reply:
OK, do you want to make note of alternative i.e creating namespace/ project during installation.

Reply:
You have taken care of this one. We can resolve it.